newa/__init__.py

```diff
@@ -458,6 +479,21 @@ class ErratumContentType(Enum):
     DOCKER = 'docker'
 
 
+def get_testing_architectures(preset: Optional[list[SysArchType]] = None) -> list[SysArchType]:
```

Comment:
Is it OK to have such a function here or would you propose a different approach?

Reply:
I have moved it as a classmethod under `SysArchType` class.

Reply:
Both are fine, I for one would put it under the `Arch` class as you did, as a "constructor", under the assumption that the class knows best how to construct its instances from various inputs.